shims.d.ts

```diff
@@ -19,3 +19,5 @@ interface NodeStream {
     read(size?: number): any;
     on(event: string | symbol, listener: (...args: any[]) => void): this;
 }
+
+interface Headers{}
```

Comment:
Why do we need an empty interface defined in here? Should it have some properties/methods defined?

Reply:
The empty interface is in case the customer doesn't have the lib: "dom" option in their code. Then this will throw an error  on compile. 
Empty interfaces is to make sure that the code compiles and merges with any existing definition.

Reply:
Thanks for the context here, much appreciated, and this now makes sense to me.